docs/index.md

```diff
@@ -92,13 +92,13 @@ installation.  If you are on a 32-bit installation, please substitute "i386" for
 First, install the Docker client binary:
 
 ```
-curl https://get.docker.com/builds/Windows/x86_64/docker-latest > /bin/docker
+$ curl -L https://get.docker.com/builds/Windows/x86_64/docker-latest > /bin/docker
 ```
 
 Next, install the Docker Machine binary:
 
 ```
-curl https://github.com/docker/machine/releases/docker-machine_windows-amd64.exe > /bin/docker-machine
+$ curl -L https://github.com/docker/machine/releases/download/v0.2.0/docker-machine_windows-amd64.exe > /bin/docker-machine
```

Comment:
@nathanleclaire yeah it'll always be `.exe`. Also downloading things to `/bin` in git shell actually makes them installed to:

    `C:\users\ahmetb\AppData\Local\VirtualStore\Program Files (x86)\Git\bin\docker-machine.exe`

it's not really convenient at all to find once user goes out msys bash. So you might wanna distribute it using Chocolatey -- and soon PackageManagement(TM) :soon: http://blogs.technet.com/b/packagemanagement/archive/2015/04/29/introducing-packagemanagement-in-windows-10.aspx